chain/jsonrpc-primitives/src/lib.rs

```diff
@@ -2,4 +2,4 @@ pub mod errors;
 pub mod message;
 pub(crate) mod metrics;
 pub mod types;
-pub(crate) mod utils;
+pub mod utils;
```

Comment:
Revert it, see my comment about `parse_params`

Reply:
reverted